pkgs/top-level/python-aliases.nix

```diff
@@ -366,6 +365,7 @@ mapAliases {
   python-bring-api = throw "python-bring-api was removed because Home Assistant switched to bring-api"; # added 2025-10-03
   python-ethtool = throw "'python-ethtool' has been renamed to/replaced by 'ethtool'"; # Converted to throw 2025-10-29
   python-igraph = throw "'python-igraph' has been renamed to/replaced by 'igraph'"; # Converted to throw 2025-10-29
+  python-jsonrpc-server = throw "python-jsonrpc-server was removed because we don't package software from fascists"; # added 2025-11-11
```

Comment:
```suggestion
  python-jsonrpc-server = throw "python-jsonrpc-server because it was no longer used by anything"; # added 2025-11-11
```

Reply:
Applied.  Also to the commit text.  And fixed the missing "has been removed".